packages/elements/src/react/common/form/index.tsx

```diff
@@ -533,6 +565,8 @@ type FormInputProps =
 const Input = React.forwardRef<React.ElementRef<typeof RadixControl>, FormInputProps>(
   (props: FormInputProps, forwardedRef) => {
     const clerk = useClerk();
+    const passkeyAutofillProp = (props as PasskeyInputProps).passkeyAutofill;
+    const signInRouterRef = SignInRouterCtx.useActorRef(true);
```

Comment:
I'm not crazy about referencing the sign in actor directly in these components, but not sure if there's another way. 🤔

Maybe @tmilewski has an idea?

Reply:
Is this better  ?
```tsx
<SignIn.Passkey>
   <Clerk.Input /> 
</SignIn.Passkey>
```

Reply:
I can see issues with that as well, because SignIn.Passkey wouldn't know whether it needs to pass the `autofill` prop or not. To avoid leaking the prop to the dom for everything else, but only pass it to Clerk.Input.

Another thought is to have a `SignIn.PasskeyInput`, but this does not feel right.

Reply:
We are now simply detecting `autoComplete="webauthn"`